Sensus.sln

```diff
@@ -30,27 +30,27 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Sensus.iOS.Shared", "Sensus
 EndProject

```

Comment:
@clmckinley Can you revert the .sln file?

Reply:
@MatthewGerber  this should be done now. Thanks!